### PR TITLE
improve grafana query

### PR DESCRIPTION
**Pull request by svenefftinge**
as suggested by @kylos101

## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: operations/observability/mixins/IDE/rules/supervisor.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
         runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/SupervisorIncomingFailuresRatioTooHigh.md
         summary: Supervisor is returning higher number of errors. This can prevent workspace usability.
       expr: |
-          sum(rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*"}[5m])) by(cluster) / sum(rate(grpc_server_handled_total{grpc_service=~"supervisor.*"}[5m])) by(cluster) > 0.01
+          sum by(cell, grpc_code, grpc_method, grpc_service, grpc_type) (rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*", cell="deutschebahn-pov"}[5m])) > 0.01
```
Comment (akosyakov) [inline operations/observability/mixins/IDE/rules/supervisor.yaml:26]: deutschebahn-pov should go away

Comment (akosyakov) [inline operations/observability/mixins/IDE/rules/supervisor.yaml:26]: we don't have cell on cloud , not sure how it would work there

Comment (kylos101) [reply operations/observability/mixins/IDE/rules/supervisor.yaml]: Good point. The transformation rule will add cell, we don't need to add here.

File: operations/observability/mixins/IDE/rules/supervisor.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ spec:
         runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/SupervisorIncomingFailuresRatioTooHigh.md
         summary: Supervisor is returning higher number of errors. This can prevent workspace usability.
       expr: |
-          sum(rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*"}[5m])) by(cluster) / sum(rate(grpc_server_handled_total{grpc_service=~"supervisor.*"}[5m])) by(cluster) > 0.01
+sum by(cell, grpc_code, grpc_method, grpc_service, grpc_type) (rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*")) > 0.01
```
Comment (kylos101) [inline operations/observability/mixins/IDE/rules/supervisor.yaml:26]: Remove cell:

```suggestion
sum by(grpc_code, grpc_method, grpc_service, grpc_type) (rate(grpc_server_handled_total{grpc_code!~"OK|Canceled", grpc_service=~"supervisor.*")) > 0.01
```

**Pull request closed without merging**